src/index.ts

```diff
@@ -105,7 +105,10 @@ export class ErrorReporting {
        request?: manualRequestExtractor.Request, additionalMessage?: string|{},
        callback?: manualInterface.Callback|{}|string) => ErrorMessage;
   event: () => ErrorMessage;
-  hapi: {register: (server: {}, options: {}, next: Function) => void};
+  hapi: {
+    register: (server: {}, options: {}, next?: Function) => void; name: string;
```

Comment:
I really hate how clang-format formats code like this. Would it be better to define a separate interface type for this?